### PR TITLE
Wait for new instances to join ecs

### PR DESCRIPTION
**Pull request by burnsed**

### CODE REVIEW

File: rollover.py [hunk 1]
```diff
@@ -230,6 +249,20 @@ def main():
...
+        # Wait for new ec2 instance to join the ECS cluster
+        #
+        if not args.scale_down and not args.noop:
+            new_asg_instances = asg.describe_instances()
+            new_ec2_id = get_added_asg_instances(asg_instances,
+                                                 new_asg_instances)[0]
+            asg_instances = new_asg_instances
+            sys.stdout.write("Waiting for replacement instance %s to join ECS..." % (new_ec2_id))
+            sys.stdout.flush()
+            while new_ec2_id not in ecs_client.list_active_ec2_instances():
```
Comment (xavi-) [inline rollover.py:261]: Nit.  Why only check one instance?

File: rollover.py [hunk 2]
```diff
@@ -230,6 +249,20 @@ def main():
...
+                time.sleep(10)
```
Comment (xavi-) [inline rollover.py:262]: Nit.  Might be worth add adding a print statement like "Still waiting..." or just "." as a bit of feedback to show the script is still working.

File: rollover.py (hunk 1 above)
Comment (burnsed) [inline rollover.py:261]: the ASG should only add one new instance when one is removed

**Comment by xavi-** [comment]
lgtm

**Pull request merged**